src/scripts/themed_posts.js

```diff
@@ -30,7 +30,8 @@ const processPosts = async function (postElements) {
   filterPostElements(postElements, { includeFiltered: true }).forEach(async postElement => {
     if (postElement.matches(blogViewSelector) && !enableOnPeepr) return;
 
-    const { blog, trail = [], content } = await timelineObject(postElement);
+    const { blog: owningBlog, authorBlog, trail = [], content, community } = await timelineObject(postElement);
+    const blog = community ? authorBlog : owningBlog;
```

Comment:
not much of a fan of renaming `blog` in the destructure here. how about...

```suggestion
    const { blog, authorBlog, trail = [], content, community } = await timelineObject(postElement);
    const visibleBlog = community ? authorBlog : blog;
```